### PR TITLE
Darkmode

### PR DESCRIPTION
**Pull request by lomesshh**
in this pr I've added dark mode to my website

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Dark mode UI looks great
Code is well structured as well
PR is good to merge

File: src/components/stylesheets/cart.css [hunk 1]
```diff
@@ -84,6 +90,11 @@
   background-color: var(--primary-color);
   color: var(--background-color);
   cursor: pointer;
+  width: 100%;
+  /* margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center; */
```
Comment (romabulani) [inline src/components/stylesheets/cart.css]: Remove the comments if not required

**Review by romabulani — APPROVED** [review]
UI is looking good even in dark mode
Good to merge

File: src/components/stylesheets/cart.css (hunk 1 above)
Comment (lomesshh) [reply src/components/stylesheets/cart.css]: done

**Pull request merged**